Libs/MRML/Widgets/qMRMLLayoutManager_p.h

```diff
@@ -150,6 +153,16 @@ public slots:
   vtkMRMLChartViewNode*   ActiveMRMLChartViewNode;
   vtkMRMLTableViewNode*   ActiveMRMLTableViewNode;
   vtkMRMLPlotViewNode*    ActiveMRMLPlotViewNode;
+
+  struct PausedView
+  {
+    std::string Name;
+    std::string NodeType;
+    bool        Paused;
```

Comment:
Maybe WasPaused would be more clear.

Reply:
I wonder if `QCache` could be used here

Reply:
@jcfr could you elaborate a little?
Do you mean if we switch WasPaused to an integer we could use QCache instead to track the paused state of all the views?

Reply:
> QCache vs integer 
One does not prevent the other. I just wanted to point out that QCache (or collection of them) could be used to manage state.

> use of integer

While i may miss some of the details, since a view can either be paused/resumed, may be simply keep track of the number of call to `pause()/resume() and increment an integer would be simpler ?